site/src/components/Footer.tsx

```diff
@@ -1,4 +1,5 @@
-import { Icon, Paragraph } from "@kaizen/component-library"
+import { Icon } from "@kaizen/component-library"
+import { Paragraph } from "@kaizen/typography"
 import classnames from "classnames"
```

Comment:
I think this might be made redundant / cause a conflict if Cass merges in her site removal, but other than that all looks good to me 👍